README.rst

```diff
@@ -2,11 +2,17 @@ XCSoar
 ======
 
 XCSoar is a tactical glide computer for Android, Linux, Mac OS X,
-Windows and Windows CE/Mobile.
+and Windows.
 
-This file is aimed at developers.  Users should read the manual which
-can be found on the `XCSoar home page <http://www.xcsoar.org/>`__.
+This file is aimed at developers. Developers should read the manual which may
+be built by running in the XCSoar source base directory
 
+make manual
```

Comment:
it's not that easy because you will likely have to install several buildtools to be able to actually run `make manual` successfully. I'm not sure the manual build instructions should be part of the README 🤔